assignment/fish-assignment.Rmd

```diff
@@ -62,46 +89,29 @@ build a tidy table with the desired columns.
  
 ```
 
-## Task 2: Mapping the Area table to marine regions
-
-In order to replicate the collapse of Atlantic Cod, 
-we need to be able to map area table from the Ram database to the marine regions. 
-
-*As an aside, this database is unclear what kind of areas the `area` table is using, they do not appear to be LMEs, EEZs, or other obvious marine region classification. Regardless, we will use them to extract the North America cod stocks.*
+I ended up with a table that had the following column names (some of which I needed to rename
+in the process!):  `country`, `areaname`, `ssb_unit`, `TC_unit`, 
+`scientificname`, `commonname`, `year`, `ssb`, and `TC`. Your columns might be different
+just make sure you contain the info listed above!
 
-Write code to pull all marine areas (listed in `ram$area`) that contain a certain substring
-in their name -- ex. "Georges Bank". 
-Hint: you want want to consider functions `filter()` or `grep()`
+## Task 2: Subsetting our data by regional id and species name 
 
-```{r}
-
-```
+In order to replicate the collapse of Atlantic Cod, we are interested in the 
+data from Atlantic Cod harvests in Canada. 
 
-We are interested in mapping the data from just the areas where Atlantic Cod are found.
-Using the table you built above, pull out distinct areas that contain
-Atlantic Cod populations into a new tidytable. 
-Hint: you may want to use functions like `filter()` or `distinct()`
+Using the table you built above, `filter()` those records of Atlantic Cod caught in
+Canada. 
 
 ```{r}
 
 ```
 
-## Task 3: Subsetting our data by regional id
-
-Using bracket notation and or the `filter()` and `pull()` functions, try pulling 
-certain subsets of ids from your table of cod areas. ex. the first 8 ids, or the ids of areas just within a certain country.
-
-Create a vector of ids of areas with Atlantic Cod and in Canada. 
 
-```{r}
-
-```
+## Task 3: Plotting Total Catch in Canada
 
+Calculate and plot the Canadian catch of Atlantic Cod (in million tons (MT)) through time using 
+your new datatable. 
 
```

Comment:
Good edits, keeps them focused on the bigger goal.  

I think the `group_by` + `summarise()` pattern is a stumbling block that takes some getting used to; not sure if more hand-holding would be good here? (though the homework reading should have done a reasonable job of this so it is probably fine like this)


I wonder if it would help to introduce grouping by a particular variable (`areaname`) as a kind of automatic alternative to filtering  and selecting the TC for each area name, one by one, and then manually adding the results?